js/main.js

```diff
@@ -0,0 +1,23 @@
+$(document).ready(function () {
+  (function appInit() {
+    // detect user has scrolled
+    $(window).scroll(function stickyNavbar() {
+      var scrollTop = $(window).scrollTop();
+      var scrolled = $('body').hasClass('scrolled');
+
+      if (!scrolled && (scrollTop > 0)) {
```

Comment:
I haven't seen any issues on Chrome, Safari or Firefox, besides, we are using a JQuery method so it will make some fallbacks there.